### DISCUSSION TITLE
AWS SSO (Google IdP) via CLI not working

### DISCUSSION THREAD

**Discussion by Larrybwoy** [aws/aws-cli#8174] [Q&A]
I have set up AWS SSO via the Google GSuite IDP and IAM Roles which allow autheticated users to connect to my environments. Everything is working properly when connecting to AWS via the Google App in the dropdown: I get prompted to select the google account to connect with after which I am prompted for an IAM Role to select.

The problem is I do not get the same behaviour when using the CLI. When running "aws sso login --profile <profilename>" it generates the code ->opens the browser to https://device.sso.eu-west-1.amazonaws.com/ -> promps me to select the google account but after I do so I am getting the famed "403. That’s an error. Error: app_not_configured_for_user" error. 
Basically it seems that I cannot reach the ACS URL " https://signin.aws.amazon.com/saml" (which is defined in the Google SSO app) if I use the CLI to connect.

Been trying to figure it out all day but running into a wall so far with what I can find online regarding this specific issue.
Any advice on this is highly appreciated!

**Comment by tim-finnigan** [comment]
Hi @Larrybwoy thanks for reaching out. Were you able to find a resolution to this issue? I'm not very familiar with Google IDP but found this documentation on that error: https://support.google.com/a/answer/6301076?hl=en

<img width="588" alt="image" src="https://github.com/aws/aws-cli/assets/87778557/4688300b-6c13-477c-bcee-7efdb04800c0">

---

Here are a couple of other resources that may be helpful:
* https://docs.aws.amazon.com/singlesignon/latest/userguide/google-idp.html
* https://aws.amazon.com/blogs/security/how-to-use-g-suite-as-external-identity-provider-aws-sso/
